pkg/costmodel/router.go

```diff
@@ -163,6 +165,22 @@ func normalizeTimeParam(param string) (string, error) {
 	return param, nil
 }
 
+// Parses the max query concurrency environment variable
+func maxQueryConcurrency() int {
+	v := os.Getenv(maxQueryConcurrencyEnvVar)
+	if v == "" {
+		return 5
+	}
+
+	result, err := strconv.Atoi(v)
+	if err != nil {
+		klog.Infof("[Warning] Failed to parse MAX_QUERY_CONCURRENCY. Defaulting to 5 - %s", err)
```

Comment:
Just a reminder that we have `log.Warningf(...)` for this stuff. A nit, but it standardizes the labeling conventions.